api/tests/acceptance/application/campaign-participation-controller_test.js

```diff
@@ -237,7 +237,7 @@ describe('Acceptance | API | Campaign Participations', () => {
         { skillId: skillIds2[2], status: 'invalidated' },
 
       ]).each((ke, id) => {
-        databaseBuilder.factory.buildKnowledgeElement({ id, userId: participant.id, assessmentId, ...ke });
+        databaseBuilder.factory.buildKnowledgeElement({ ...ke, id, userId: participant.id, assessmentId, });
```

Comment:
Si les merges font ce type de probleme, est-ce qu'il ne faudrait pas éviter d'utiliser la déconstruction et bien indiquer ce que l'on souhaite faire passer ?

Reply:
la déconstruction c'est la façon moderne de merger quand même puisque c'est niveau platforme et built-in dans le langage. Faut simplement qu'on s'habitue à l'utiliser ;)

Reply:
de plus la quasi totalité des erreurs que j'ai corrigé c'était moi qui savais pas l'utiliser ;) je resolve!